diff --git a/node_modules/@astrojs/image/components/Image.astro b/node_modules/@astrojs/image/components/Image.astro
index dea492d..9f83b01 100644
--- a/node_modules/@astrojs/image/components/Image.astro
+++ b/node_modules/@astrojs/image/components/Image.astro
@@ -6,7 +6,7 @@ import type { ImageComponentLocalImageProps, ImageComponentRemoteImageProps } fr
 
 export type Props = ImageComponentLocalImageProps | ImageComponentRemoteImageProps;
 
-const { loading = 'lazy', decoding = 'async', ...props } = Astro.props;
+const { decoding = 'async', ...props } = Astro.props;
 
 if (props.alt === undefined || props.alt === null) {
 	warnForMissingAlt();
@@ -15,4 +15,4 @@ if (props.alt === undefined || props.alt === null) {
 const attrs = await getImage(props);
 ---
 
-<img {...attrs} {loading} {decoding} />
+<img {...attrs} {decoding} />
