diff --git a/node_modules/astro/dist/assets/services/service.js b/node_modules/astro/dist/assets/services/service.js
index c048754..f4d4869 100644
--- a/node_modules/astro/dist/assets/services/service.js
+++ b/node_modules/astro/dist/assets/services/service.js
@@ -107,8 +107,8 @@ const baseService = {
       ...attributes,
       width: targetWidth,
       height: targetHeight,
-      loading: attributes.loading ?? "lazy",
-      decoding: attributes.decoding ?? "async"
+      loading: attributes.loading,
+      decoding: attributes.decoding
     };
   },
   getSrcSet(options) {
