diff --git a/dist/assets/services/service.js b/dist/assets/services/service.js
index c0487549ac684ec6a844f0a339563627a8f2e77a..f4d48696e953f81572595b19878a6f257a18ea66 100644
--- a/dist/assets/services/service.js
+++ b/dist/assets/services/service.js
@@ -107,8 +107,8 @@ const baseService = {
       ...attributes,
       width: targetWidth,
       height: targetHeight,
-      loading: attributes.loading ?? "lazy",
-      decoding: attributes.decoding ?? "async"
+      loading: attributes.loading,
+      decoding: attributes.decoding
     };
   },
   getSrcSet(options) {
