diff --git a/node_modules/astro/dist/assets/services/service.js b/node_modules/astro/dist/assets/services/service.js
index 3e84d1e..afd8faf 100644
--- a/node_modules/astro/dist/assets/services/service.js
+++ b/node_modules/astro/dist/assets/services/service.js
@@ -85,8 +85,8 @@ const baseService = {
       ...attributes,
       width: targetWidth,
       height: targetHeight,
-      loading: attributes.loading ?? "lazy",
-      decoding: attributes.decoding ?? "async"
+      loading: attributes.loading,
+      decoding: attributes.decoding
     };
   },
   getSrcSet(options) {
